convert-hf-to-gguf.py

```diff
@@ -249,6 +249,9 @@ def write_tensors(self):
             if name.endswith((".attention.masked_bias", ".attention.bias", ".rotary_emb.inv_freq")):
                 continue
 
+            if name.startswith("bert."):
+                name = name.removeprefix("bert.")
+                
```

Comment:
Again, I think this should be done in the `BertModel` class instead of the `Model` class, but in `modify_tensors` instead of `write_tensors`.

(also I think `.startswith()` is redundant with `.removeprefix()`, which already only removes the prefix when the string starts with it)